### DISCUSSION TITLE
Clickable Row with clickable elements

### DISCUSSION THREAD

**Discussion by matt-jolt** [TanStack/table#2924] [Q&A]
I couldn't find anything in the docs or examples on this after quite some time so if I've missed it I apologize.

I have the following table:
<img width="316" alt="Screen Shot 2020-12-15 at 3 45 20 PM" src="https://user-images.githubusercontent.com/62953316/102281690-a49eb300-3eec-11eb-93d9-d57d841dc819.png">

Each row is clickable as well as having a checkbox and a menu (not in the picture).
How on earth do I get the `onClick` for the row to not execute when I click on the checkbox or menu?

**Comment by gargroh** [comment]
stop event propagation

**Reply by matt-jolt** [reply]
On the checkbox click right? I tried that there and it didn't change anything.

**Comment by wnharper** [comment]
I'm having this same issue. I've stopProgation on the checkbox as well as the row onclick handler and still I'm not able to get the checkboxes or other links/buttons in the row to work.
@matt-jolt did you find a solution?

**Reply by matt-jolt** [reply]
@wnharper  Yeah. In the `onClick` function we just added some kind of logic to check what is being clicked.
We check the class of the target (or we've also used the column id), and base the handler of off that.

We never used the stop propogation pattern, just do nothing if its not an identifier we want.

**Reply by wnharper** [reply]
Thanks @matt-jolt , I appreciate you getting back to me!
